.github/mergify.yml

```diff
@@ -1,7 +1,9 @@
 pull_request_rules:
   - name: Add needs-dco label when DCO check failed
     conditions:
-      - base=master
+      - or:
+        - base=master
+        - base~=2+(.\d+){2}$
```

Comment:
Maybe should be 
```
(\.\d+)
```

Reply:
Any reason？ I have tested on my repo & it works for 2.0.2.